src/test/kotlin/io/bazel/kotlin/builder/Deps.java

```diff
@@ -82,19 +83,52 @@ public Dep build() {
       }
     }
 
-    /** Collect all of the compile jars of all the dependencies. */
+    /**
+     * Collect all of the compile jars of all the dependencies.
+     */
     public static Stream<String> classpathOf(Dep... dependencies) {
       return Stream.of(dependencies).flatMap(it -> it.compileJars().stream());
     }
 
-    /** Import a single dep. Similar to a `kt_jvm_import` or a `kt_js_import`. */
+    /**
+     * Import a single dep. Similar to a `kt_jvm_import` or a `kt_js_import`.
+     */
     public static Dep importJar(String label, String compileJar) {
       return Dep.builder()
           .label(label)
           .compileJars(
               ImmutableList.of(BazelRunFiles.resolveVerified(compileJar).getAbsolutePath()))
           .build();
     }
+
+    /**
+     * Reads dependency path from jvm_args for a given label, provided that the label has been added
+     * as a jvm property.
+     * <p>
+     * See src/test/kotlin/io/bazel/kotlin/defs.bzl for an example of proper label and runfile
+     * passing.
+     *
+     * @param label The label of the resource expected to be included
+     * @return Dep reprenseting the resource
+     * @throws IllegalArgumentException if the label does not exist.
+     */
+    protected static Dep fromLabel(String label) {
+      // jvm properties do not allow slashes or :.
+      String key = label.replaceAll("/", ".").replaceAll(":", ".");
+      Properties properties = System.getProperties();
+      Preconditions.checkArgument(properties.containsKey(key),
+          String.format("Unable to find %s in properties:\n%s", key,
```

Comment:
It is so painful that this is in Java, since we end up doing all this nonsense, instead of...

```
require(properties.containsKey(key)) {
    "Unable to find $key in properties:\n${properties.keySet().joinToString("\n") { it.toString() }}"
}
```

Reply:
TBH, I'm not sure why there is so much kotlin/java running around. I'd prefer it had one langauge.